source/includes/v2/_catalog.md

```diff
@@ -104,167 +108,170 @@ curl 'https://documentation-resources.opendatasoft.com/api/v2/catalog/datasets?l
+curl 'https://documentation-resources.opendatasoft.com/api/v2/catalog/datasets/?select=sum(records_count)&group_by'
 ```
 
 > API Response
 
 ```json
 {
-    "aggregations": [
-        {
-            "sum(records_count)": 105936
+    "datasets": [
+        {"dataset":
+            {
+                "sum(records_count)": 105891
+            }
         }
     ]
 }
 ```
 
-> Aggregation with an multiple group_by
+> Grouped results by grouping with multiple metadata
 
 ```shell
-curl 'https://documentation-resources.opendatasoft.com/api/v2/catalog/aggregates/?select=features,theme,count(*)&group_by=features,theme&limit=5'
+curl 'https://documentation-resources.opendatasoft.com/api/v2/catalog/datasets/?select=features,theme,count(*)&group_by=features,theme&limit=5'
 ```
 
 > API Response
 
 ```json
 {
-    "aggregations": [
-        {
-            "theme": "Administration, Government, Public finances, Citizenship",
-            "count(*)": 1,
-            "features": "analyze"
+    "datasets": [
+        {"dataset":
+            {
+                "theme": "Administration, Government, Public finances, Citizenship",
+                "count(*)": 1,
+                "features": "analyze"
+            },
         },
-        {
-            "theme": "Culture, Heritage",
-            "count(*)": 2,
-            "features": "analyze"
+        {"dataset":
+            {
+                "theme": "Culture, Heritage",
+                "count(*)": 2,
+                "features": "analyze"
+            },
         },
-        {
-            "theme": "Economy, Business, SME, Economic development, Employment",
-            "count(*)": 2,
-            "features": "analyze"
+        {"dataset":
+            {
+                "theme": "Economy, Business, SME, Economic development, Employment",
+                "count(*)": 2,
+                "features": "analyze"
+            },
         },
-        {
-            "theme": "Environment",
-            "count(*)": 1,
-            "features": "analyze"
+        {"dataset":
+            {
+                "theme": "Environment",
+                "count(*)": 1,
+                "features": "analyze"
+            },
         },
-        {
-            "theme": "Product",
-            "count(*)": 1,
-            "features": "analyze"
+        {"dataset":
+            {
+                "theme": "Product",
+                "count(*)": 1,
+                "features": "analyze"
+            }
         }
     ]
 }
 ```
 
-This endpoint provides an aggregation facility in the datasets catalog.
 
-An aggregation query returns a JSON array containing an object for each group created by the query.
-Each JSON object contains a key/value pair for each `select` instruction.
-However, without the `group_by` parameter, the query returns an array with only one object.
+By defaut, this endpoint returns a list of datasets metadata, one item of the list containing metadata of a given dataset.
```

Comment:
defaut -> default

I am note sure of the corresponding grammar rule, but I would write "dataset metadata" or "dataset's metadata"